charts/ingress-nginx/templates/default-backend-deployment.yaml

```diff
@@ -4,13 +4,13 @@ kind: Deployment
 metadata:
   labels:
     {{- include "ingress-nginx.labels" . | nindent 4 }}
-    app.kubernetes.io/component: {{ .Values.defaultBackend.name | quote }}
-  name: {{ template "ingress-nginx.defaultBackend.fullname" . }}
+    app.kubernetes.io/component: default-backend
+  name: {{ include "ingress-nginx.defaultBackend.fullname" }}
```

Comment:
missing the period for the include on these

Reply:
Thanks.